### PR TITLE
Feature(MGX-241): Configs cleanup and Helmfile environment deployments

### PR DESCRIPTION
**Pull request by tenequm**

### CODE REVIEW

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -224,7 +224,7 @@ jobs:
 
   run-benchmarks:
     name: Run runtime benchmarks
-    runs-on: [performance]
+    runs-on: [self-hosted, performance]
```
Comment (mateuszaaa) [inline .github/workflows/build-and-test.yml:227]: runtime benchmarks should be only run on specific hardware configuration matching what parity uses, i thing self-hosted are stronger ones, right ?

File: devops/dockerfiles/node-builder/Dockerfile [hunk 2]
```diff
@@ -1,71 +0,0 @@
-#
```
Comment (mateuszaaa) [inline devops/dockerfiles/node-builder/Dockerfile:1]: where is it now ?

File: devops/dockerfiles/node-new/Dockerfile [hunk 3]
```diff
@@ -1,11 +0,0 @@
-FROM debian:buster
```
Comment (mateuszaaa) [inline devops/dockerfiles/node-new/Dockerfile:1]: where is it now ?

File: devops/dockerfiles/node/Dockerfile [hunk 4]
```diff
@@ -1,6 +1,11 @@
-FROM mangatasolutions/node-runner:0.2
-ARG NODE_BINARY
-ARG WASM
-COPY $NODE_BINARY /mangata/node
-COPY $WASM /mangata/
-ENTRYPOINT ["/mangata/node"]
+FROM debian:buster
```
Comment (mateuszaaa) [inline devops/dockerfiles/node/Dockerfile:1]: could you point reference to dockerfile to builder image ?

Comment (mateuszaaa) [reply devops/dockerfiles/node/Dockerfile:1]: buster & buster its fine :+1:

**Review by mateuszaaa — APPROVED** [review]
lgtm

**Review by vanderian — APPROVED** [review]

**Pull request merged**